File: modules/common-model/src/test/java/com/shterneregen/securelan/common/model/CommonModelSmokeTest.java
package com.shterneregen.securelan.common.model;

import org.junit.jupiter.api.Test;

import java.time.Instant;

import static org.junit.jupiter.api.Assertions.assertEquals;

class CommonModelSmokeTest {
    @Test
    void shouldCreateChatMessage() {
        ChatMessage message = new ChatMessage("msg-1", "alice", "bob", "hello", Instant.now());
        assertEquals("hello", message.content());
    }
}

File: modules/common-model/src/test/kotlin/com/shterneregen/securelan/common/model/CommonModelSmokeTest.kt
package com.shterneregen.securelan.common.model

import org.junit.jupiter.api.Assertions.assertEquals
import org.junit.jupiter.api.Test
import java.time.Instant

class CommonModelSmokeTest {
    @Test
    fun shouldCreateChatMessage() {
        val message = ChatMessage("msg-1", "alice", "bob", "hello", Instant.now())

        assertEquals("hello", message.content)
    }
}
